docs/demo.md

```diff
@@ -0,0 +1,77 @@
+
+Files that can be used in this demo are found in the demo/ folder in the root of this repository. These files are simulated fastq, and bam-files that represents a father/mother/child trio. The files cover a 2000 bp region of the CFTR gene, with 200 paired reads for each file 2*151bp. The fastq files were generated using [wgsim](https://github.com/lh3/wgsim), and the bam files were generated aligning these fastq-files with [bwa](https://github.com/lh3/bwa) against the [GRCh37](https://www.ensembl.org/info/website/tutorials/grch37.html) reference. The child has a simulated de-novo heterozygous SNV 7:117235031 G->A. This demo shows how this family, along with the causative clinical variant can be uploaded into the mutacc database, and later exported to create a new synthethic sample.
+
+in this demo, the flag ```-d/--demo``` is used after the main command , i.e. ```mutacc --demo [subcommands] [options]```. With this flag, no configuration file will be necessary. This will create the mutacc root directory in the home directory of the user ```~/mutacc_root_demo/``` which can be removed after the demo is made. For this demo to work, it requires that a mongodb process is running on host 'localhost' and 27017. If mutacc is installed on a conda environment (which is recommended!) source the environment before starting this demo. Don't forget to ```cd```into the root directory of this repo.
+
+## Extract reads from case
+
+Before this family can be uploaded into the database, the reads from overlapping the given causative variant (specified in ```demo/variant1.vcf.gz```) must be extracted. This is done with the following command
+
+ ```terminal
+ mutacc --demo extract --case demo/case.yaml --padding 100
+ ```
+
+have a look at the ```demo/case.yaml``` file. Here all paths to bam-files and the vcf containing
+the causative variant are given, along with other meta-data.
+
+This will extract all reads spanning the variant position with an additional padding
+of 100 bp on both sides. The variant reads will be placed in ```~/mutacc_root_demo/reads/demo_trio/<sample>/<date>/``` as fastq.gz files. There will also be a new json formatted file in ```~/mutacc_root_demo/imports/demo_trio_import_mutacc.json``` that can now be uploaded into the database.
+
+## Upload case to the database
+
+The upload is done with one simple command
+
+```terminal
+mutacc --demo db import ~/mutacc_root_demo/imports/demo_trio_import_mutacc.json
+```
+
+The information in the .json file will now get imported into the database
+
+## Query the database
+
+Congrats! there is now one case and one variant in the database. These can now be queried for
+with the command
+
+```terminal
+mutacc --demo db export --case-query '{}' --member child --proband --sample-name child
+```
+
+Let's go through the options given in this example.
+
+--case-query '{}': This option will take a json-formated string. When making queries
```

Comment:
Yes, I think you are right. I will change this sometime in the future (perhaps not in this PR) to be a little more intuitive. because I think it is a bit more flexible when testing the program.